app/config/collections.php

```diff
@@ -931,43 +970,61 @@
                 'filters' => [],
             ],
             [
-                '$id' => ID::custom('store'),
+                '$id' => ID::custom('schedule'),
                 'type' => Database::VAR_STRING,
                 'format' => '',
-                'size' => 256,
+                'size' => 100,
                 'signed' => true,
                 'required' => false,
                 'default' => null,
                 'array' => false,
                 'filters' => [],
             ],
             [
-                '$id' => ID::custom('hostname'),
+                '$id' => ID::custom('active'),
+                'type' => Database::VAR_BOOLEAN,
+                'signed' => true,
+                'size' => 0,
+                'format' => '',
+                'filters' => [],
+                'required' => false,
+                'default' => null,
+                'array' => false,
+            ],
+            [
+                '$id' => ID::custom('region'),
                 'type' => Database::VAR_STRING,
                 'format' => '',
-                'size' => 256,
+                'size' => 10,
                 'signed' => true,
-                'required' => false,
+                'required' => true,
                 'default' => null,
                 'array' => false,
                 'filters' => [],
-            ]
+            ],
         ],
         'indexes' => [
             [
-                '$id' => ID::custom('_key_project'),
+                '$id' => ID::custom('_key_region_resourceType_resourceUpdatedAt'),
                 'type' => Database::INDEX_KEY,
-                'attributes' => ['projectInternalId'],
-                'lengths' => [Database::LENGTH_KEY],
-                'orders' => [Database::ORDER_ASC],
+                'attributes' => ['region', 'resourceType', 'resourceUpdatedAt'],
```

Comment:
Where can I see the query which corresponds to this index?

Reply:
@Meldiron let's add to the backlog or fix now if you have time.